doc/devel/contributing.rst

```diff
@@ -37,8 +37,15 @@ particularly suited for new contributors by the label
 These are well documented issues, that do not require a deep understanding of
 the internals of Matplotlib. The issues may additionally be tagged with a
 difficulty. ``Difficulty: Easy`` is suited for people with little Python experience.
-``Difficulty: Medium`` and ``Difficulty: Hard`` are not trivial to solve and
-require more thought and programming experience.
+``Difficulty: Medium`` and ``Difficulty: Hard`` require more programming experience.
+This could be for a variety of reasons, among them:
```

Comment:
maybe add something like "uses advanced features of the Python language" ?  We have a bunch of decorators and context managers in the library which are easy to use, but the implementation is subtle.

Reply:
You might want to double-check my phrasing. I tried to keep super neutral, but informative.